src/geointerface.jl

```diff
@@ -110,6 +71,11 @@ let pointtypes = (wkbPoint, wkbPoint25D, wkbPointM, wkbPointZM),
         return getcoorddim(geom)
     end
 
+    GeoInterface.x(::GeometryTraits, geom::AbstractGeometry) = getx(geom, 0)
+    GeoInterface.y(::GeometryTraits, geom::AbstractGeometry) = gety(geom, 0)
+    GeoInterface.z(::GeometryTraits, geom::_AbstractGeometryZ) = getz(geom, 0)
+    GeoInterface.m(::GeometryTraits, geom::_AbstractGeometryM) = getm(geom, 0)
```

Comment:
Can we add unit tests for these lines?

Reply:
All tested except `m`, as I'm not sure we can even construct any `_AbstractGeometryM` objects, and I've pushed that to another PR so this can merge.